packages/types/versions.test-d.ts

```diff
@@ -0,0 +1,45 @@
+import { Octokit } from "@octokit-next/core";
+import { expectType } from "tsd";
+
+import "@octokit-next/types-rest-api-ghes-3.0";
+
+export async function run() {
+    const octokit = new Octokit({
+        version: "ghes-3.1",
+    });
+
+    const response = await octokit.request("GET /ghes-only");
+
+    expectType<boolean>(response.data.ok);
+
+    const OctokitGHES31 = Octokit.withDefaults({
+        version: "ghes-3.1",
```

Comment:
A fun little tip for debugging: when mouse-hovering over the `version` in each of these calls, if it comes up as `string` then TypeScript isn't being told it needs to specify the generic for the function call. If it comes up as the string literal such as `"ghes-3.1"` here then TypeScript is storing it as the specific literal.